changelog/_unreleased/2025-11-25-plugin-migration-indexer.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
we do not use the changelog files anymore. see this ADR https://github.com/shopware/shopware/blob/trunk/adr/2025-10-28-changelog-release-info-process.md

instead you can add a small information to the release info https://github.com/shopware/shopware/blob/trunk/RELEASE_INFO-6.7.md#core